README.md

```diff
@@ -273,7 +273,9 @@ There is a complete runnable example [here](examples/collection-format.js).
 
 <a name="hide"></a>
 ### Hide a route
-Sometimes you may need to hide a certain route from the documentation, just pass `{ hide: true }` to the schema object inside the route declaration.
+Sometimes you may need to hide a certain route from the documentation, there is 2 alternatives:
+- just pass `{ hide: true }` to the schema object inside the route declaration.
+- use the tag `X-HIDDEN` inside the route declaration.
```

Comment:
Could you put this value as input in config of the plugin?

```
fastify.register(require('../index'), {
  swagger: {
    host: 'localhost',
    schemes: ['http'],
    consumes: ['application/json'],
    produces: ['application/json']
  },
  exposeRoute: true,
  hiddenTag: 'X-HIDDEN'
})
```

I say that because I spot an open discussion on that https://github.com/OAI/OpenAPI-Specification/issues/433

The default value is ok

Reply:
Done. Also changed the doc and the typing.